frontend/src/views/PurchasePage.vue

```diff
@@ -346,6 +369,9 @@ export default {
         COMMERCE_EQUITABLE: { text: "Commerce équitable" },
         PERFORMANCE: { text: "Produits acquis sur la base de leurs performances en matière environnementale" },
         EQUIVALENTS: { text: "Produits équivalents aux produits bénéficiant de ces mentions ou labels" },
+        FRANCE: { text: "Provenance France" },
+        SHORT_DISTRIBUTION: { text: "Circuit-court" },
+        LOCAL: { text: "Local (spécifier plus)" },
```

Comment:
```suggestion
        LOCAL: { text: "Local" },
```
Pas sur que "spécifier plus" soit nécessaire, surtout que le champ apparaît tout de suite après et est obligatoire.

Reply:
Je pensais que p-e ça rassure les utilisateurs qu'il n'y aura pas des questions additionnels pour les autres aussi